doc/source/examples.rst

```diff
@@ -1,154 +1,11 @@
-Code examples
-=============
+Example guides
```

Comment:
Maybe we should find a different name for this. It looks a bit weird in this context:

<img width="279" alt="image" src="https://github.com/adap/flower/assets/423981/e847f71a-9ac4-4b88-bbf1-3a673c550c9e">

Reply:
Maybe 'Walkthroughs' would be better? Should we also change 'Setup guides'?

Reply:
@danieljanes thoughts?